web/src/emoji_picker.ts

```diff
@@ -326,7 +342,7 @@ function filter_emojis() {
     }
 }
 
-function toggle_reaction(emoji_name, event) {
+function toggle_reaction(emoji_name: string, event: JQuery.ClickEvent | JQuery.KeyDownEvent): void {
```

Comment:
This function handles two code paths. One with `event` being a `KeyDown`
```
$(".emoji-popover-filter").on("keydown", process_enter_while_filtering);
```
and other being a `Click` event.